src/app/public/modules/action-buttons/action-buttons.component.spec.ts

```diff
@@ -71,6 +75,7 @@ describe('StacheActionButtonsComponent', () => {
         RouterTestingModule
       ],
       providers: [
+        SkyAuthTokenProvider,
```

Comment:
i would expect this to be handled by `SkyRestrictedViewModule`, which id being brought in via `StacheActionButtonsModule` right?

Reply:
I thought so too, but it complained about not having the provider. Let me look into whats going on...

Reply:
It looks like `SkyRestrictedViewModule` doesn't export `SkyAuthTokenProvider`. It probably should as its injected into it's own auth service here: https://github.com/blackbaud/skyux-lib-restricted-view/blob/master/src/app/public/modules/restricted-view/restricted-view-auth.service.ts#L22 , right?

Reply:
that sounds right. sounds like the module structure isn't ideal at the moment. separate pr to fix that maybe?

Reply:
Done. I've removed this from the providers.